mammoth/docx/body_xml.py

```diff
@@ -71,18 +71,26 @@ def _create_reader(numbering, content_types, relationships, styles, docx_file, f
 
     def text(element):
         return _success(documents.Text(_inner_text(element)))
-
+   
+    def element_or_value(element):
+        if element and ("w:val" in element.attributes):
+            element_value = element.attributes["w:val"]
+            if element_value == 'false':
+                return False
+            elif element_value == 'true':
+                return True
+        return element
 
     def run(element):
         properties = element.find_child_or_null("w:rPr")
         vertical_alignment = properties \
             .find_child_or_null("w:vertAlign") \
             .attributes.get("w:val")
         
-        is_bold = properties.find_child("w:b")
-        is_italic = properties.find_child("w:i")
-        is_underline = properties.find_child("w:u")
-        is_strikethrough = properties.find_child("w:strike")
+        is_bold = element_or_value(properties.find_child("w:b"))
+        is_italic = element_or_value(properties.find_child("w:i"))
+        is_underline = element_or_value(properties.find_child("w:u"))
+        is_strikethrough = element_or_value(properties.find_child("w:strike"))
```

Comment:
The naming of this function is a little odd: we just want to get the Boolean value of the element. Getting the element itself isn't helpful to us. Perhaps instead something like `read_boolean_element()`?